cross-platform/react-app/package.json

```diff
@@ -128,5 +128,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "relativeDependencies": {
+    "@itwin/mobile-sdk-core": "../../../mobile-sdk-core",
```

Comment:
what happens when these dont exist?

Reply:
These are only used by [relative-deps](https://github.com/mweststrate/relative-deps), so everything functions as normal.

Reply:
@toddsouthenbentley, that seems reasonable, but did you test by checking out your branch into a clean directory that doesn't have the other packages alongside?

Reply:
Yes